src/app/screens/connectors/ChooseConnectorPath/index.tsx

```diff
@@ -1,66 +1,135 @@
+                <Button tabIndex={-1} label={t("other.connect")} primary flex />
+              </Link>
             }
-            actions={<ConnectAlby />}
           />
           <ConnectorPath
-            title={t("other.title")}
-            description={t("other.description")}
+            title={t("voltage.title")}
+            icon={<img src="assets/icons/voltage.png" className="w-10 h-10" />}
+            description={t("voltage.description")}
             content={
+              <>
+                <FeatureItem type="success">{t("voltage.point1")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point2")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point3")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point4")}</FeatureItem>
+              </>
+            }
+            actions={
               <Link
-                to="choose-connector"
-                className="flex flex-wrap gap-6 w-72 lg:w-80 mx-auto my-5 items-center"
-                tabIndex={-1}
+                to="https://info.getalby.com/voltage"
+                target="_blank"
+                rel="noreferrer noopener"
+                className="flex flex-1"
               >
-                {connectorRoutes.slice(1, 8).map(({ path, title, logo }) => (
-                  <img
-                    key={path}
-                    title={title}
-                    src={logo}
-                    alt="logo"
-                    className="inline rounded-lg w-12"
-                  />
-                ))}
-                <p className="text-gray-500 dark:text-neutral-400">
-                  {t("other.and_more")}
-                </p>
+                <Button
+                  tabIndex={-1}
+                  label={t("voltage.connect")}
+                  primary
+                  flex
+                />
               </Link>
             }
-            actions={
-              <Link to="choose-connector" className="flex flex-1">
-                <Button tabIndex={-1} label={t("other.connect")} primary flex />
-              </Link>
+          />
+          <ConnectorPath
+            title={t("alby.title")}
+            icon={
+              <img
+                src="assets/icons/alby.png"
+                className="w-10 h-10 rounded-md"
+              />
+            }
+            description={t("alby.description")}
+            content={
+              <>
+                <FeatureItem type="warning">{t("alby.point1")}</FeatureItem>
+                <FeatureItem type="warning">{t("alby.point2")}</FeatureItem>
+                <FeatureItem type="success">{t("alby.point3")}</FeatureItem>
+                <FeatureItem type="success">{t("alby.point4")}</FeatureItem>
+              </>
             }
+            actions={<ConnectAlby />}
           />
         </div>
       </div>
     </div>
   );
 }
+
+const FeatureItem = ({
```

Comment:
is this missing some light mode font config? 
there the text is white.

Reply:
Great catch! :sweat: